pkl-core/src/main/java/org/pkl/core/util/MathUtils.java

```diff
@@ -63,7 +63,7 @@ public static boolean isMathematicalInteger(double x) {
    * Long.bitCount(Long.MIN_VALUE) == 1}, but {@link Long#MIN_VALUE} is not a power of two.
    */
   public static boolean isPowerOfTwo(long x) {
-    return x > 0 & (x & (x - 1)) == 0;
+    return x > 0 && (x & (x - 1)) == 0;
```

Comment:
This actually has the same logic; bitwise AND and logical AND have the same meaning for booleans.

But, it probably makes sense to change this for style reasons. Also, bitwise AND will not short-circuit so that's an added bonus.

Reply:
yes